src/setup/serverless_config.go

```diff
@@ -108,15 +115,34 @@ func (s *Serverless) AddFunctionConfig(subex *SubExperiment, index int) {
 // RemoveService removes the service defined in serverless.yml
 func RemoveService() string {
 	slsRemoveCmd := exec.Command("sls", "remove")
-	slsRemoveCmd.Dir = "src/setup/deployment/raw-code/serverless/aws"
+	slsRemoveCmd.Dir = "setup/deployment/raw-code/serverless/aws"
 	slsRemoveMessage := util.RunCommandAndLog(slsRemoveCmd)
 	return slsRemoveMessage
 }
 
 // Deploys the functions defined in the serverless.com file
 func deployService() string {
 	slsDeployCmd := exec.Command("sls", "deploy")
-	slsDeployCmd.Dir = "src/setup/deployment/raw-code/serverless/aws"
+	slsDeployCmd.Dir = "setup/deployment/raw-code/serverless/aws"
 	slsDeployMessage := util.RunCommandAndLog(slsDeployCmd)
 	return slsDeployMessage
 }
+
+func GetAWSLambdaRuntimeIdentifierUsingLanguageName(language string) string {
+	supportedRuntimes := map[string]string{
+		"Python": "python3.10",
+		"Java":   "java17",
+		"Go":     "go1.x",
+	}
+	runtime, ok := supportedRuntimes[language]
+	if !ok {
+		log.Fatalf("Unable to get AWS Lambda runtime identifier for the langauge %s", language)
+	}
+	return runtime
+}
+
+func buildGoBinary(functionImageName string) {
```

Comment:
Awesome that you managed to get it runnig! I was struggling with Go deployment for like a day. However, I would wait a bit till the boiler plate gets approved (it does not look much different from what we have now). In the boiler plate, I have designeted a new package where all the build modules will reside.